docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - "5678:5678"
     volumes:
       - ".:/app"
+      - "/app/client/dist"
```

Comment:
Won't this break using npm on the host?

Reply:
With using `npm` for development on host,  `npm start` on host for development.

Reply:
I meant, wouldn't setting up the volume this way make the changes on the host not seen in the container?

Reply:
I see. 
I'll update the changes made on the host can appear  in the container.

Reply:
I'm sorry, I misunderstood.
This Pull Request close.